### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by bencbernstein**

### CODE REVIEW

File: Apiary/ViewControllers/HiveViewController.swift [hunk 1]
```diff
@@ -55,6 +55,11 @@ final class HiveViewController: UIViewController {
         }
       }
     }
+
```
Comment (OLI9292) [inline Apiary/ViewControllers/HiveViewController.swift]: why so many empty lines?

File: Apiary/ViewControllers/HiveViewController.swift [hunk 2]
```diff
@@ -164,13 +170,15 @@ final class HiveViewController: UIViewController {
     hex.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(rootHexTapped(_:))))
     hiveView.addSubview(hex)
     
-    if (rootLevel == nil) || (rootLevel?.locked == true) { hex.alpha = 0.5 }
+    if (rootLevel == nil) || (rootLevel?.locked == true) {hex.alpha = 0.5 }
```
Comment (OLI9292) [inline Apiary/ViewControllers/HiveViewController.swift]: this space can be added back

File: Apiary/ViewControllers/HiveViewController.swift [hunk 3]
```diff
@@ -195,6 +203,38 @@ final class HiveViewController: UIViewController {
...
+      $0.clipsToBounds = true
+      $0.alpha = 0
+      $0.backgroundColor = .white
+      $0.textAlignment = .center
+      $0.font = Text.large(style: .regular).font
+      self.view.addSubview($0)
+    }
+
+    guard
+      let user = RealmManager.user,
```
Comment (OLI9292) [inline Apiary/ViewControllers/HiveViewController.swift]: shouldn't grab user from here.  either User.fetch - or in this case you can grab the last `rootLevels` from the view model

File: Apiary/ViewControllers/HiveViewController.swift [hunk 4]
```diff
@@ -195,6 +203,38 @@ final class HiveViewController: UIViewController {
...
+      let currentLevel = user.currentLockedRootLevel
+      else { return }
+    
+    delay(seconds: 0.5) {
+      UIView.animate(withDuration: 0.2, animations: { 
+        rootLabel.alpha = 1
+        rootLabel.typewriter(string: "\(currentLevel.name.uppercased()) Unlocked!", duration: 0.04, completion: {
```
Comment (OLI9292) [inline Apiary/ViewControllers/HiveViewController.swift]: this isn't an UIView animation so it shouldn't be inside an animations block

File: Apiary/Views/Intermission.swift [hunk 5]
```diff
@@ -170,12 +170,17 @@ class Intermission: UIView {
     let rootLabel = UILabel().then {
       addSubview($0)
       $0.alpha = 0
-      $0.text = User.fetch.unlockedRootLevels.last?.name.uppercased()
-      $0.textColor = Palette.honey.color
-      $0.font = Text.extraExtraLarge(style: .regular).font
+      if let lastRootLevel = User.fetch.unlockedRootLevels.last {
+        let rootPassed = NSMutableAttributedString(string: "ROOT PASSED", attributes: [NSForegroundColorAttributeName: UIColor.black, NSFontAttributeName: Text.large(style: .regular).font as Any])
```
Comment (OLI9292) [inline Apiary/Views/Intermission.swift]: all caps looks kind of harsh imo, maybe smaller text below the root

**Comment by bencbernstein** [comment]
I'll incorporate feedback to do a new correct answer animation. I'll create a hive animation for unlocking a root

File: Apiary/Views/Intermission.swift (hunk 5 above)
Comment (bencbernstein) [inline Apiary/Views/Intermission.swift]: will lower case

**Pull request merged**